js-weather/script.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const selectElement = document.getElementById('select-cityname');
+const url = 'https://api.openweathermap.org/data/2.5/weather';
+let params = { appid: '4b5774e9f3d2a07b84f0f2f88e486224', q: 'London', units: 'metric', lang: 'ja' };
+
+function parseResponse(data) {
```

Comment:
(修正不要)
アロー関数使うとよいですね。
```suggestion
const parseResponse = (data) => {
```